tests/pages/externalpages/pocket/base.py

```diff
@@ -0,0 +1,94 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from pypom import Page, Region
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+
+
+class BaseRegion(Region):
+    pass
+
+
+class BasePage(Page):
+    def __init__(self, selenium, base_url, locale="en-US", **url_kwargs):
+        super(BasePage, self).__init__(selenium, base_url, locale=locale, **url_kwargs)
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda s: self.seed_url in s.current_url)
+        el = self.find_element(By.TAG_NAME, "html")
+        self.wait.until(lambda s: "loaded" in el.get_attribute("class"))
+
+        # Dismiss cookie prompt before running tests
+        cookie_prompt = self.wait.until(lambda s: s.find_element(By.ID, "onetrust-accept-btn-handler"))
+        self.wait.until(lambda s: expected.element_to_be_clickable(cookie_prompt))
+        cookie_prompt.click()
+        self.wait.until(lambda s: cookie_prompt.is_displayed() is False)
+
+        return self
```

Comment:
I think the problem here is due to the cookie banner animating (CSS animations are notoriously tricky for Selenium to understand). The code here fails for me locally, at least intermittently. It's tricky, because we don't seem (?) to control the code/HTML for this popup since it's third party. If we did, then we could have added a CSS class once the animation finishes.

I tried this which seems to help a little, worth a shot pushing this for a test run?
```suggestion
        # Dismiss cookie prompt before running tests
        self.wait.until(lambda s: expected.presence_of_element_located(s.find_element(By.ID, "onetrust-banner-sdk")))
        cookie_banner = self.find_element(By.ID, "onetrust-banner-sdk")
        self.wait.until(lambda s: "animation-name" not in cookie_banner.get_attribute("style"))

        accept_cookie_button = self.find_element(By.ID, "onetrust-accept-btn-handler")
        self.wait.until(lambda s: expected.element_to_be_clickable(accept_cookie_button))
        accept_cookie_button.click()

        self.wait.until(lambda s: cookie_banner.is_displayed() is False)
        
        return self
```